test/cases/upgrade8to9.sh

```diff
@@ -103,7 +103,7 @@ sudo virt-install --name rhel-test \
                   --memory 3072 \
                   --vcpus 2 \
                   --disk size=20 \
-                  --location http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.8.0/compose/BaseOS/x86_64/os/ \
+                  --location http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.9.0/compose/BaseOS/x86_64/os/ \
```

Comment:
Shouldn't these actually be the 8.9 GA repository ?

Reply:
That's a good question. It depends on what we're actually testing here, I thought we're testing 8.10 nightly > 9.4 nightly upgrade path in which case I made a mistake and it should be 8.10 actually... However if we're testing latest RHEL-8-GA > latest RHEL-9-nightly then it should be GA here instead. I tried searching for an old ticket we had for this but I can't find this exact information anywhere, I'll ask the LEAPP team on slack.

Reply:
I've ammended the related upgrade commit with the link to the slack thread and it's using the latest rhel-8 nightly as base now.